ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -1585,11 +1593,7 @@ private CadTemplate readPolyline3D()
 			//Flags RC 70 NOT DIRECTLY THE 70. Bit-coded (76543210):
 			//0 : Closed(70 bit 0(1))
 			//(Set 70 bit 3(8) because this is a 3D POLYLINE.)
-			pline.Flags |= PolylineFlags.Polyline3D;
-			if ((this._objectReader.ReadByte() & 1U) > 0U)
-			{
-				pline.Flags |= PolylineFlags.ClosedPolylineOrClosedPolygonMeshInM;
-			}
+			bool closed = (this._objectReader.ReadByte() & 1U) > 0U;
```

Comment:
Changing the method readPolyline3D() seems to be outside of the scope for this task please revert the changes, unless there is a major reason to change this method.